tests/src/core/getExports.js

```diff
@@ -4,7 +4,7 @@ import sinon from 'sinon';
 import eslintPkg from 'eslint/package.json';
 import typescriptPkg from 'typescript/package.json';
 import * as tsConfigLoader from 'tsconfig-paths/lib/tsconfig-loader';
-import ExportMap from '../../../src/ExportMap';
+import ExportMap from '../../../src/exportMapBuilder';
```

Comment:
```suggestion
import Exports from '../../../src/exportMapBuilder';
```
should this be named "Exports" or "ExportMapBuilder"?

Reply:
Fair.

- https://github.com/import-js/eslint-plugin-import/pull/2985/commits/cc1b19e6417c5f28ba2ade6db6477a24a8f42b3b